lib/td/command/table.rb

```diff
@@ -23,6 +24,9 @@ def table_create(op)
       end
       type = s.to_sym
     }
+    op.on('--include-v BOOLEAN', TrueClass, 'set include_v flag') {|v| params[:include_v] = v}
+    op.on('--detect-schema BOOLEAN', TrueClass, 'set detect schema flag') {|v| params[:detect_schema] = v}
+    op.on('--expire-days DAYS', Integer, 'set table expiration') {|v| params[:expire_days] = v}
```

Comment:
Don't need to validate negative number as well as table_update?
https://github.com/treasure-data/td/pull/196/files#diff-aa873830df56d6da16838ad8cf7f68f3R352